src/server.test.ts

```diff
@@ -18,6 +18,30 @@ jest.mock('./api/contributionsApi', () => {
         getAmpExperimentData: jest.fn().mockImplementation(() => Promise.resolve({})),
     };
 });
+jest.mock('./tests/amp/ampEpicTests', () => {
+    return {
+        getAmpExperimentData: jest.fn().mockImplementation(() => Promise.resolve({})),
+    };
+});
+jest.mock('./tests/banners/bannerDeployCache', () => {
+    return {
+        bannerDeployCaches: {
+            contributions: jest.fn().mockImplementation(() => Promise.resolve({})),
+            subscriptions: jest.fn().mockImplementation(() => Promise.resolve({})),
+        },
+    };
+});
+
+jest.mock('./tests/banners/ChannelBannerTests', () => {
+    return {
+        channel1BannersAllTestsGenerator: jest
+            .fn()
+            .mockImplementation(() => () => Promise.resolve([])),
+        channel2BannersAllTestsGenerator: jest
+            .fn()
+            .mockImplementation(() => () => Promise.resolve([])),
+    };
+});
```

Comment:
Is this bit also required to get the tests to pass?

Reply:
Yes, without it the tests fail because it's trying to get aws credentials. I think before this change to `isDev` it was breaking in a different way, that didn't cause the tests to fail but still caused warnings.

Reply:
Oh nice - great to get a better fix in place!